doonut-app-external-api/src/main/kotlin/com/doonutmate/member/service/strategy/StrategyMemberDelete.kt

```diff
@@ -0,0 +1,7 @@
+package com.doonutmate.member.service.strategy
+
+import com.doonutmate.member.controller.dto.DeleteRequest
+
+interface StrategyMemberDelete {
```

Comment:
[Kotline Coding converntions](https://kotlinlang.org/docs/coding-conventions.html#choose-good-names)에 맞춰 클래스명은 명사 또는 명사구로 짓는게 좋을 것 같습니다.

> The name of a class is usually a noun or a noun phrase explaining what the class is: List, PersonReader.
```suggestion
interface MemberDeleteStrategy {
```